test/test_variable_fps_video.py

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import json
+import pathlib
+import pytest
+import numpy as np
+import torch
+
+from torchcodec._core import (
+    create_from_file,
+    add_video_stream,
+    get_next_frame,
+    get_frame_at_index,
+    get_frame_at_pts,
+    get_frames_in_range,
+    get_json_metadata,
+    seek_to_pts,
+)
+
+from .utils import (
+    assert_frames_equal,
+    cpu_and_cuda,
+    TestVideo,
+    TestVideoStreamInfo,
+    TestFrameInfo,
+    _get_file_path,
+    VAR_FPS_VIDEO,
+)
+
+class TestVariableFPSVideoDecoder:
+    def _check_video_exists(self):
+        try:
+            VAR_FPS_VIDEO.path
+        except FileNotFoundError:
+            pytest.skip("Variable FPS test video not found")
```

Comment:
Since we're checking the test file into the repo, I don't think we should skip if the file isn't found. The test should just fail, like the other unit tests do.